index.html

```diff
@@ -3,89 +3,81 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="css/style.css">
-    <title>Book store!</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
CSS file is not properly linked and not loaded at all

Reply:
I do not know why CSS is not loaded even though I write 'link' tag the same way I wrote it today every time when I do a project or webpage and CSS loads normally after pushing it to Github , I can not show you some examples if that is needed

Reply:
Sorry I forgot that Pourija has put style.css in css folder that is why it did not load at all, my mistake.

Reply:
Good observation why it doesn't work.